libs/datasets/timeseries.py

```diff
@@ -395,8 +396,8 @@ def _merge_attributes(df1: pd.DataFrame, df2: pd.DataFrame) -> pd.DataFrame:
     if missing_columns:
         _log.warning(f"Re-adding empty columns: {missing_columns}")
         wide = wide.reindex(columns=[*wide.columns, *missing_columns])
-    # Make all non-GEO_DATA_COLUMNS numeric so that aggregation functions work on them.
-    numeric_columns = list(all_columns - set(GEO_DATA_COLUMNS))
+    # Make all numeric columns numeric so that aggregation functions work on them.
```

Comment:
"Make columns expected to be numeric have a numeric dtype so that aggregation functions work on them."